js/lessonfilter.js

```diff
@@ -52,6 +52,50 @@ function applySortFromURI(uri,featureList) {
   featureList.sort(sortType, { order: sortOrder });
 }
 
+function lunrSearch(searchString, idx, corpus, featureList) {
+  const results = idx.search(searchString);
+  var docs = results.filter(result => corpus.some(doc => result.ref === doc.url)).map(result => {
+    let doc = corpus.find(o => o.url === result.ref);
+    return {
+      ...result,
+      ...doc
+    };
+  });
+
+  const BUFFER = 30 // Number of characters to show for kwic-results
+  const MAX_KWIC = 3
+  docs.map((doc) => {
+    let elementName = '/'+doc.url.split('/').slice(3, ).join('/')
```

Comment:
wonder if there's a more reliable way to get the lesson language than parsing the url